lms/views/api/grading.py

```diff
@@ -156,20 +156,7 @@ def get_speedgrader_launch_url(self):
             "resource_link_id": parsed_params.get("resource_link_id"),
         }
 
-        if parsed_params.get("document_url"):
```

Comment:
These are the parameters read from our "Submission API" that the backend itself sets and get forwarded by the frontend.

They get serialized to query params and sent to Canvas.